server/webhook_jira.go

```diff
@@ -28,6 +76,47 @@ type JiraWebhook struct {
 	IssueEventTypeName string `json:"issue_event_type_name"`
 }
 
+// toEventEnums converts a JiraWebhook struct into a slice of internal event identifiers
```

Comment:
Is this description accurate now? Maybe: "map of internal event identifiers"?

Reply:
Thanks for pointing this out, the comment is certainly incorrect now. In my progress of converting the legacy webhook code to use these, I am realizing that maps should be used in favor of arrays only when we plan on mutating the structure after creation, or if the structure is big enough that we would want an O(1) lookup.

I actually don't see how/why we would be using the result returned here as something we want to mutate, and the dataset of event types will always be of negligible size, so I am siding with reverting back to returning an array here instead. What are your thoughts?

Reply:
Just in general I think maps are pretty lightweight in Go, and we shouldn't worry about that kind of thing. I think use whatever is easiest to program if there's no concern about time complexity.